bin/taskcluster-ssh.js

```diff
@@ -0,0 +1,73 @@
+var taskcluster = require('taskcluster-client');
+var Promise = require('promise');
+var DockerClient = require('docker-exec-websocket-server').DockerExecClient;
+var https = require('https');
+var url = require('url');
+var _ = require('lodash');
+var assert = require('assert');
+require('../lib/config').load();
+
+var INTERACTIVE_SOCKET_NAME = 'private/docker-worker/interactive.sock';
+
+var yargs = require('yargs')
+  .usage('SSH into task container')
+  .example('taskcluster ssh', 'taskId')
+
+var taskId = yargs.argv._[0];
+
+var queue = new taskcluster.Queue();
+
+queue.status(taskId).then(function(result) {
+  // Get run and check state
+  var run = _.last(result.status.runs);
+  console.log("Latest run: " + run.runId);
+  if (run.state !== 'running') {
+    console.log("Task must be running for us to connect!");
+    process.exit(1);
+  }
+
+  // Build URL for the interactive socket
+  var signedUrl = queue.buildSignedUrl(
+    queue.getArtifact, taskId, run.runId, INTERACTIVE_SOCKET_NAME
+  );
+
+  var req = https.get(url.parse(signedUrl));
+  return new Promise(function(accept, reject) {
+    req.on('response', accept);
+    req.on('error', reject);
+  });
+}).then(function(res) {
+  assert(res.statusCode === 303, "Expected a 303 redirect");
+  var socketUrl = res.headers.location;
+
+  var client = new DockerClient({
+    url: socketUrl,
+    tty: true,
+    command: ['/.taskclusterutils/busybox', 'sh'],
+  });
+
+  return client.execute().then(function() {
+    // Set terminal size and update when terminal is resized
+    client.resize(process.stdout.rows, process.stdout.columns);
+    process.stdout.on('resize', function() {
```

Comment:
I'm not sure if I'm doing something wrong on my end, but when I resized my terminal, it doesn't look like it adjusted it on the receiving side.  Top pane here is the docker websocket session, and below that is an ssh session.  Notice that $MANIFEST is cut off in the top pane.
![screen shot 2015-08-10 at 10 50 58 am](https://cloud.githubusercontent.com/assets/2592630/9175944/ec9a82a0-3f4d-11e5-929d-bae7f9d945b0.png)